src/Jackett.Common/Utils/SizeUtil.cs

```diff
@@ -0,0 +1,55 @@
+using System.Text.RegularExpressions;
```

Comment:
I believe there is already a size utility see src/Jackett.Common/Models/ReleaseInfo.cs
for example
https://github.com/Jackett/Jackett/blob/483a72babd7cbf73d0ec9add8624e7feb054d3f3/src/Jackett.Common/Indexers/TVVault.cs#L132

Reply:
Ah I was expecting it to be somewhere in utils if it existed. Converted over.